e2e-prepare.sh

```diff
@@ -6,12 +6,19 @@ echo "(1/7) Running preflight checks and cleanups..."
 
 if [ -z "$PYTHON" ];
 then
-  echo "  > Error: Please set the PYTHON environment variable."
-  exit
+  echo "  > Python env-var not found, attempting detection via which"
+  PYTHON="$(which python2.7)"
+fi
+
+if [ -z "$PYTHON" ];
+then
+  echo "  > ERROR: You need to install python2.7 to compile node-gyp related dependencies."
 else
   echo "  > Found Python at: $PYTHON"
 fi
 
+exit
```

Comment:
Didn't run this yet but.... Are you sure that this goes here and not in line 15 inside the*then*?

Reply:
This shouldn't be here, lol.